programs/create-genesis/main.cpp

```diff
@@ -58,6 +59,8 @@ void config_reader::set_program_options(options_description& cli) {
             "the location of the genesis info file (absolute path or relative to the current directory).")
         ("output-file,o", bpo::value<bfs::path>(&out_file)->default_value("cyberway-genesis.dat"),
             "the file to write generic genesis data to (absolute or relative path).")
+        ("ee-output-file,e", bpo::value<bfs::path>(&ee_file)->default_value("events-genesis.dat"),
```

Comment:
`make_absolute` not called for this. And check another things not called (if exist).

Reply:
Thanks. Fixed.